var/spack/repos/builtin/packages/py-greenlet/package.py

```diff
@@ -10,6 +10,7 @@ class PyGreenlet(PythonPackage):
     """Lightweight in-process concurrent programming"""
 
     homepage = "https://github.com/python-greenlet/greenlet"
-    url      = "https://pypi.io/packages/source/g/greenlet/greenlet-0.4.13.tar.gz"
+    url      = "https://github.com/python-greenlet/greenlet/archive/0.4.17.tar.gz"
```

Comment:
Can you explain why this URL change is necessary? It changes the checksum of the previous version. Also see https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#url

Reply:
good point. looks like this was a bad call. i will revert this.

Reply:
fixed this